projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitActivity.java

```diff
@@ -340,6 +282,52 @@ private void checkPermissions() {
     }
   }
 
+  // Pops a share dialog
+  void sharePdf() {
```

Comment:
This can be moved to the summery fragment since you are only using it there

Reply:
The thought is to have it belong to the LawsuitActivity, as it might come in handy to give the user another chance to save the file before he/she proceed to claiming the lawsuit online. (In case they didn't save the file at the summary fragment. They need it at נט המשפט).

So I left as part of `LawsuitActivity`'s "operations" for now. Let me know if you think otherwise.